src/test/mapper/generic.mapper.spec.ts

```diff
@@ -0,0 +1,54 @@
+import {User} from "../../modules/domains/custom/user/user.domain";
+import {Name} from "../../modules/DTO/value-object/name";
+import {Surname} from "../../modules/DTO/value-object/surname";
+import {Hashmap} from "../../modules/DTO/hashmap";
+import {GenericMapper} from "../../modules/mapper/generic.mapper";
+import {ComplexDomain} from "./complex.domain";
+import {InvalidDomain} from "./invalid.domain";
+
+describe("GenericMapper test", () => {
```

Comment:
These describes are very important because they tell us what is being tested within it.  In this case, if we are only testing the getter methods of the domain, I would change it for something like "Testing getter methods of the Generic Mapper"

Reply:
In this case the describe is for all the functionalities and use cases about GenericMapper, I think it's okay.
Nevertheless as I'm going to remove GenericMapper the describe will be different.